ios/README.md

```diff
@@ -5,11 +5,22 @@
 * iOS 8+
 * SwiftLint (`brew install swiftlint`)
 
-### Compiling from Command line
+## Keyman App
+
+This is the iOS app which is distributed. The source for the app is found in **keyman/**. This project only contains
+the views specific to the Keyman app, while the core functionality is found in KeymanEngine.framework.
+
+### Building
 To build Keyman for iOS, please run the build.sh build script within this folder.
-You will need to run with the command-line argument `-no-codesign` or establish alternate code-signing permissions to complete the build for the final app.
+You will need to run with the command-line argument `-no-codesign` or establish alternate code-signing permissions to
+complete the build for the final app.
 
-To run the app within a simulator after build.sh is completed, "Run" the Keyman scheme/target within Xcode.
+To run the app within a simulator after build.sh is completed,
+1. Open **keyman/Keyman/Keyman.xcodeproj** in Xcode.
+2. "Run" the Keyman target within Xcode.
```

Comment:
Maybe bold **Keyman** to make the target name clearer?

Reply:
In our READMEs, we usually have bold to denote a path and elsewhere in the readme, the target names aren't bolded so I think it's best to leave it unbolded.